api/graphql/schema.py

```diff
@@ -15,7 +17,14 @@ def Schema(models):
                     args={},
                     resolve=get_test_field,
                     description="Test field",
-                )
+                ),
+                "challenges": challenges_field(models),
+            },
+        ),
+        mutation=GraphQLObjectType(
+            name="Mutation",
+            fields={
+                "createWorkweekHustle": create_workweek_hustle_field(models),
```

Comment:
This adds a mutation called `createWorkweekHustle` in the GraphQL schema, which executes the `create_workweek_hustle_field` function.